lookout/style/format/cmdline_tools.py

```diff
@@ -173,6 +175,24 @@ def add_parser(name, help):
     rule_parser.add_argument("model", help="Path to the model file.")
     rule_parser.add_argument("hash", help="Hash of the rule (8 chars).")
 
+    # Should be removed as soon as https://github.com/src-d/style-analyzer/issues/557 resolved.
+    calc_expected_vnodes = add_parser("calc-expected-vnodes-number",
+                                      "Get a csv file with an expected number of nodes set. "
```

Comment:
Write the CSV file with expected numbers of virtual nodes extracted from repositories.